src/Symfony/Component/Routing/Tests/Generator/Dumper/PhpGeneratorDumperTest.php

```diff
@@ -84,6 +84,33 @@ public function testDumpWithRoutes()
         $this->assertEquals('/app.php/testing2', $relativeUrlWithoutParameter);
     }
 
+    public function testDumpWithLocalizedRoutes()
+    {
+        $this->routeCollection->add('test.en', (new Route('/testing/is/fun'))->setDefault('_locale', 'en'));
+        $this->routeCollection->add('test.nl', (new Route('/testen/is/leuk'))->setDefault('_locale', 'nl'));
+
+        $code = $this->generatorDumper->dump([
+            'class' => 'LocalizedProjectUrlGenerator',
+        ]);
+        file_put_contents($this->testTmpFilepath, $code);
```

Comment:
You should `unlink()` the fixtures at end of this test.

Reply:
already handled in tearDown